src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUserResponse.java

```diff
@@ -2,23 +2,33 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
 import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUserStatus;
 
 public class ProfessionalUserResponse {
 
     @JsonProperty
-    private final String id;
+    private UUID userIdentifier;
+    @JsonProperty
+    private String firstName;
+    @JsonProperty
+    private String lastName;
     @JsonProperty
-    private final String firstName;
+    private String email;
     @JsonProperty
-    private final String lastName;
+    private ProfessionalUserStatus status;
     @JsonProperty
-    private final String email;
+    private List<String> roles = new ArrayList<String>();
 
     public ProfessionalUserResponse(ProfessionalUser user) {
-        this.id = user.getId().toString();
+        this.userIdentifier = user.getUserIdentifier();
```

Comment:
Do you need check for null? which we had the problem on SuperUserResponse.java

Reply:
Ahh yes, I will include the same piece of code. Once again, good catch!